controller/array_action/array_mediator_ds8k.py

```diff
@@ -235,7 +211,7 @@ def create_volume(self, name, size_in_bytes, capabilities, pool_id, volume_prefi
         try:
             cli_kwargs = {}
             cli_kwargs.update({
-                'name': shorten_volume_name(name, volume_prefix),
```

Comment:
I think `volume_prefix` can now be removed altogether (including mediator interface)

Reply:
because its also from mediator interface I think its better for after GA